docs.html

```diff
@@ -58,6 +58,17 @@ <h1>Quick start</h1>
             <pre>
     <code class="html">
 <link rel="stylesheet" href="https://github.com/jalajcodes/max-ui/blob/dev/components/max.css"/>
+    </code>
+    </pre>
+            <h4>Fonts and Icons</h4>
+            <p>
+              To use the fonts and icons used in the component library add these tag inside your
+              <code>head</code> tag.
+            </p>
+            <pre>
+    <code class="html">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Fira+Code&family=Poppins:wght@400;500;600&display=swap" />
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css" />
```

Comment:
you can also define css links in root directory so don't have to write them down everywhere

Reply:
will do that